test/test_datapipe.py

```diff
@@ -1372,19 +1410,22 @@ def test_simple_traverse(self):
 
     # TODO(VitalyFedyunin): This test is incorrect because of 'buffer' nature
     # of the fork fake implementation, update fork first and fix this test too
-    @skipIfNoDill
-    def test_traverse_forked(self):
-        numbers_dp = NumbersDataset(size=50)
-        dp0, dp1, dp2 = numbers_dp.fork(3)
-        dp0_upd = dp0.map(lambda x: x * 10)
-        dp1_upd = dp1.filter(lambda x: x % 3 == 1)
-        combined_dp = dp0_upd.mux(dp1_upd, dp2)
-        graph = torch.utils.data.graph.traverse(combined_dp)
-        expected = {combined_dp: {dp0_upd: {dp0: {}}, dp1_upd: {dp1: {}}, dp2: {}}}
-        self.assertEqual(expected, graph)
+    # TODO: This conflicts with new .fork implementation "TypeError: cannot pickle 'generator' object"
```

Comment:
We need to fix it in this PR

Reply:
I had another look but haven't fix this test yet. I am not 100% sure what the graph is doing, I will look again next week.

Reply:
Fixed.